[2023-06-28T09:30:44.113+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-28T09:29:00+00:00/task_id=create_select permission to 509
[2023-06-28T09:30:44.114+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-28T09:29:00+00:00/task_id=create_select permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-28T09:29:00+00:00/task_id=create_select'
[2023-06-28T09:30:44.172+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-28T09:29:00+00:00/task_id=create_select permission to 509
[2023-06-28T09:30:44.173+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-28T09:29:00+00:00/task_id=create_select permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-28T09:29:00+00:00/task_id=create_select'
[2023-06-28T09:30:44.221+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 2_Test_Badge_5_Loop.create_select scheduled__2023-06-28T09:29:00+00:00 [queued]>
[2023-06-28T09:30:44.242+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 2_Test_Badge_5_Loop.create_select scheduled__2023-06-28T09:29:00+00:00 [queued]>
[2023-06-28T09:30:44.243+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-06-28T09:30:44.263+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): create_select> on 2023-06-28 09:29:00+00:00
[2023-06-28T09:30:44.271+0000] {standard_task_runner.py:57} INFO - Started process 1456 to run task
[2023-06-28T09:30:44.276+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '2_Test_Badge_5_Loop', 'create_select', 'scheduled__2023-06-28T09:29:00+00:00', '--job-id', '453', '--raw', '--subdir', 'DAGS_FOLDER/test_loop.py', '--cfg-path', '/tmp/tmprw1entte']
[2023-06-28T09:30:44.277+0000] {standard_task_runner.py:85} INFO - Job 453: Subtask create_select
[2023-06-28T09:30:44.347+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-28T09:29:00+00:00/task_id=create_select permission to 509
[2023-06-28T09:30:44.348+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-28T09:29:00+00:00/task_id=create_select permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-28T09:29:00+00:00/task_id=create_select'
[2023-06-28T09:30:44.352+0000] {task_command.py:410} INFO - Running <TaskInstance: 2_Test_Badge_5_Loop.create_select scheduled__2023-06-28T09:29:00+00:00 [running]> on host a3cd9226e3f8
[2023-06-28T09:30:44.544+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='2_Test_Badge_5_Loop' AIRFLOW_CTX_TASK_ID='create_select' AIRFLOW_CTX_EXECUTION_DATE='2023-06-28T09:29:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-06-28T09:29:00+00:00'
[2023-06-28T09:30:44.546+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-28T09:30:44.548+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cd /dbt && dbt run --model toto_select']
[2023-06-28T09:30:44.567+0000] {subprocess.py:86} INFO - Output:
[2023-06-28T09:30:48.997+0000] {subprocess.py:93} INFO - [0m09:30:48  Running with dbt=1.5.1
[2023-06-28T09:30:53.017+0000] {subprocess.py:93} INFO - [0m09:30:53  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2023-06-28T09:30:53.018+0000] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2023-06-28T09:30:53.019+0000] {subprocess.py:93} INFO - - models.badge_5.time_of_day_lu
[2023-06-28T09:30:53.097+0000] {subprocess.py:93} INFO - [0m09:30:53  Found 3 models, 0 tests, 0 snapshots, 0 analyses, 457 macros, 0 operations, 1 seed file, 0 sources, 0 exposures, 0 metrics, 0 groups
[2023-06-28T09:30:53.103+0000] {subprocess.py:93} INFO - [0m09:30:53
[2023-06-28T09:30:55.286+0000] {subprocess.py:93} INFO - [0m09:30:55  Concurrency: 200 threads (target='dev')
[2023-06-28T09:30:55.288+0000] {subprocess.py:93} INFO - [0m09:30:55
[2023-06-28T09:30:55.345+0000] {subprocess.py:93} INFO - [0m09:30:55  1 of 1 START sql table model ***.toto_select ............................... [RUN]
[2023-06-28T09:30:57.119+0000] {subprocess.py:93} INFO - [0m09:30:57  1 of 1 OK created sql table model ***.toto_select .......................... [[32mSUCCESS 1[0m in 1.77s]
[2023-06-28T09:30:57.192+0000] {subprocess.py:93} INFO - [0m09:30:57
[2023-06-28T09:30:57.194+0000] {subprocess.py:93} INFO - [0m09:30:57  Finished running 1 table model in 0 hours 0 minutes and 4.09 seconds (4.09s).
[2023-06-28T09:30:57.241+0000] {subprocess.py:93} INFO - [0m09:30:57
[2023-06-28T09:30:57.242+0000] {subprocess.py:93} INFO - [0m09:30:57  [32mCompleted successfully[0m
[2023-06-28T09:30:57.243+0000] {subprocess.py:93} INFO - [0m09:30:57
[2023-06-28T09:30:57.245+0000] {subprocess.py:93} INFO - [0m09:30:57  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2023-06-28T09:30:58.077+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-06-28T09:30:58.114+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=2_Test_Badge_5_Loop, task_id=create_select, execution_date=20230628T092900, start_date=20230628T093044, end_date=20230628T093058
[2023-06-28T09:30:58.203+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-28T09:30:58.249+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
